app/models/agents/twitter_user_agent.rb

```diff
@@ -64,6 +67,10 @@ def validate_options
       if options[:include_retweets].present? && !%w[true false].include?(options[:include_retweets])
         errors.add(:base, "include_retweets must be a boolean value string (true/false)")
       end
+      
+      if options[:exclude_replies].present? && !%w[true false].include?(options[:exclude_replies])
```

Comment:
Of course, no problem!

Existing agents won't have the `:exclude_replies` option yet, so it will be `nil`.  `options[:exclude_replies].present?` will return `true`, but `%w[true false].include?(options[:exclude_replies])` will be false.

I think you could just skip this validation.